sources/packages/web/src/views/aest/AESTDashboard.vue

```diff
@@ -1,3 +1,5 @@
 <template>
-  <formio-container formName="aestdashboard"> </formio-container>
+  <full-page-container :full-width="true" layout-template="centered">
```

Comment:
I guess that `layout-template="centered"` is not needed here.

Reply:
its needed, if not given we see 2 boxed from the formio panel. Also its similar to the insitutionDashboard.